cmd/bnscli/clitests/keygen.test

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+# Use a custom key path just in case the host is using original one.
+tempdir=`mktemp -d`
+keypath=$tempdir/key.priv
+export BNSCLI_PRIV_KEY=$keypath
+
+bnscli keygen
+
+# Key generation in non deterministic (cryptography 101) so we cannot compare
+# its value. We can only ensure that enough bytes was generated.
+echo "generated private key length: `base64 $keypath | wc -c`"
```

Comment:
The 90 in the result was confusing until I realized this was base64 encoded.
Why not just `wc -c < $keypath` ? which should be 64 (or 66 if there is the protobuf header)